components/mgmtworker/scripts/snapshot_permissions_fixer

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+restservice_user=$(cat /usr/lib/systemd/system/cloudify-restservice.service | grep User | sed 's/User=//')
```

Comment:
This is true, and that relationship... apparently doesn't exist. Update coming shortly.